hicommonwealth/commonwealth
# Address management chart

## Link to Issue
Closes: #8709

## Description of Changes
- Introduced a "Manage Wallet Addresses" section on the edit profile page.
- Added an interactive table displaying wallet addresses and their associated communities.
- Enabled functionality to add and remove wallet addresses.
- Implemented "..." menu with additional options for each wallet address.
- Dynamic table expansion for multiple wallet addresses and connected communities.

## Test Plan
- Navigate to the Edit Profile page.
- Go to the new "Manage Addresses" section.
- Verify the following:
  - The new UI is displayed correctly with wallet addresses on the left and associated communities on the right.
  - The "..." menu appears for each wallet address and shows the disconnect options when clicked.
  - Removing a wallet address:
    -  Click on the "..." menu for a wallet address.
    - Select the "Disconnect Address" option.
    - Confirm that the wallet address is removed from the table.

## Deployment Plan


## Other Considerations

Figma Link : https://www.figma.com/design/N7vGFBW0XNQG5lgUIflObX/Address-Mgmt?node-id=223-12578&t=TO2EMFwyebMRpS74-0
